junit5/src/test/java/io/specto/hoverfly/junit5/HoverflyCoreStaticTestInstanceTest.java

```diff
@@ -0,0 +1,35 @@
+package io.specto.hoverfly.junit5;
+
+import io.specto.hoverfly.junit.core.Hoverfly;
+import io.specto.hoverfly.junit.core.HoverflyMode;
+import io.specto.hoverfly.junit5.api.HoverflyCore;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(HoverflyCoreResolver.class)
+public class HoverflyCoreStaticTestInstanceTest {
+
+    @HoverflyCore
+    public static Hoverfly hoverfly;
+
+    @Test
+    public void shouldInjectHoverflyAndStartItAutomatically() {
+        assertThat(hoverfly.getHoverflyInfo()).isNotNull();
+    }
+
+    @Test
+    public void shouldInjectHoverflyParameterBeingTheSameAsStatic(@HoverflyCore(mode = HoverflyMode.CAPTURE) Hoverfly hoverfly) {
```

Comment:
So `@HoverflyCore(mode = HoverflyMode.CAPTURE)` is ignored here? If you need to change Hoverfly mode for this method, you would need to call this: `hoverfly.setMode(HoverflyMode.CAPTURE);`

Reply:
No, in this case you set hoverfly in `CAPTURE` mode.